### ISSUE TITLE
Software detection

### ISSUE THREAD

**Issue by JeroenVinke** [monterey-framework/monterey#236]
Can we detect visual studio code, atom and others so that we can add app launchers automatically?

**Comment by adriatic** [comment]
This is normally done by inspecting the registry (on Windows platform, that is) - like searching for `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio`

To solve this right we would have to make a platform independent implementation, something we are not supposed to do now. I would propose to use Monterey's own registry of tiles instead and encourage users to configure these tiles themselves. We should provide examples for that in https://aurelia-ui-toolkits.gitbooks.io/monterey-overview/content/monterey_tiles/app_launcher.html

**Comment by JeroenVinke** [comment]
Yeah, you should be able to install the app launchers, and somehow set one as your preferred editor / browser.
